x-pack/plugins/actions/server/application/connector/methods/get_all/get_all.ts

```diff
@@ -63,27 +63,29 @@ export async function getAll({
     })),
   ].sort((a, b) => a.name.localeCompare(b.name));
 
-  mergedResult.forEach((connector) => {
+  const connectors = await injectExtraFindData({
+    kibanaIndices: context.kibanaIndices,
+    scopedClusterClient: context.scopedClusterClient,
+    connectors: mergedResult,
+  });
+
+  connectors.forEach((connector) => {
     // Try to validate the connectors, but don't throw.
     try {
-      connectorSchema.validate(connector);
+      connectorWithExtraFindDataSchema.validate(connector);
```

Comment:
I realised that we don't validate the whole connector data we return here. 
`referencedByCount` field is added by `injectExtraFindData`